homework/index.js

```diff
@@ -29,16 +33,126 @@
     });
     return elem;
   }
+  function addOptions(select, repositories) {
+    repositories.forEach(repo => createAndAppend('option', select, { text: repo.name }));
+  }
+
+  function addContributorsDiv(url, container) {
+    fetchJSONPromise(url)
+      .then(data => {
+        const rightDiv = createAndAppend('div', container, { class: 'rightDiv', id: 'rightDiv' });
+        createAndAppend('p', rightDiv, {
+          text: 'Contributors',
+          class: 'rightDivHeader',
+        });
+        const ul = createAndAppend('ul', rightDiv, { class: 'contUl' });
+        data.forEach(cont => {
+          const li = createAndAppend('li', ul, { class: 'contLi' });
+          const contDiv = createAndAppend('div', li, { class: 'contDiv' });
+          createAndAppend('img', contDiv, {
+            src: cont.avatar_url,
+            class: 'contImg',
+          });
+          createAndAppend('div', contDiv, {
+            src: cont.html_url,
```

Comment:
A `div` does not support an `src` attribute. This is what the HTML validator says:

```
Error: Attribute “src” not allowed on element “div” at this point.
```